monthly-contributions/src/test/scala/com/gu/support/workers/errors/PayPalErrorsSpec.scala

```diff
@@ -1,46 +1,41 @@
 package com.gu.support.workers.errors
 
-import java.io.ByteArrayOutputStream
-
 import com.gu.config.Configuration
 import com.gu.okhttp.RequestRunners
 import com.gu.paypal.PayPalService
 import com.gu.support.config.PayPalConfig
-import com.gu.support.workers.Fixtures.{createPayPalPaymentMethodJson, wrapFixture}
 import com.gu.support.workers.LambdaSpec
-import com.gu.support.workers.exceptions.RetryUnlimited
-import com.gu.support.workers.lambdas.CreatePaymentMethod
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
 
 class PayPalErrorsSpec extends LambdaSpec with MockWebServerCreator with MockServicesCreator {
-  "Timeouts from PayPal" should "throw a NonFatalException" in {
-    val createPaymentMethod = new CreatePaymentMethod(timeOutServices)
-
-    val outStream = new ByteArrayOutputStream()
-
-    a[RetryUnlimited] should be thrownBy {
-      createPaymentMethod.handleRequest(wrapFixture(createPayPalPaymentMethodJson()), outStream, context)
-    }
-  }
-
-  "500s from PayPal" should "throw a NonFatalException" in {
-    val server = createMockServer(500, "Uh Oh!")
-    val baseUrl = server.url("/v1")
-    val services = errorServices(baseUrl.toString)
-
-    val createPaymentMethod = new CreatePaymentMethod(services)
-
-    val outStream = new ByteArrayOutputStream()
-
-    a[RetryUnlimited] should be thrownBy {
-      createPaymentMethod.handleRequest(wrapFixture(createPayPalPaymentMethodJson()), outStream, context)
-    }
-
-    // Shut down the server. Instances cannot be reused.
-    server.shutdown()
-  }
+  //  "Timeouts from PayPal" should "throw a NonFatalException" in {
```

Comment:
Is this intentionally left commented out?

Reply:
Nope, well spotted!